### ISSUE TITLE
What should the protocolLoader do when the provided protocol isn't valid? (2)

### ISSUE THREAD

**Issue by wwqrd** [codaco/Network-Canvas#168]
We need to handle the case that the protocol file specified isn't valid.

**Depends on #192**

# Tasks

- [ ] Implement error dialog for protocol loader (1)
- [ ] Restart protocol loading when error is dismissed (1)

**Comment by jthrilly** [comment]
Show an error dialog, and restart the flow (whatever it is).

**Comment by jthrilly** [comment]
We should perhaps make a new issue around implementing a dialog box in with the correct UI styling. Let's discuss this later.

**Comment by jthrilly** [comment]
# Total estimate: 1.5

- Implement error dialog for protocol loader (1)
- Restart protocol loading when error is dismissed (0.5)

**Comment by rebeccamadsen** [comment]
Estimate (2)
--

- Implement error dialog for protocol loader (1)
- Restart protocol loading when error is dismissed (1)
